### DISCUSSION TITLE
SAS與微軟結盟 用AI與雲端技術提前預測水災

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS3073-C/Python-Community-GS3073-C#133] [General]
SAS舉美國北卡羅萊納州的卡瑞鎮（Cary）為例，目前正使用由微軟和SAS聯手推出的物聯網服務，進行一個重要的水災預測解決方案，藉由使用感測器、氣候資料、SAS 物聯網分析以及 Azure IoT 平台，預計能提高對於河川水位上升的狀態警覺，預測那裡可能發生水災，以及透過自動化功能改善我們的緊急應變能力。

不可能說連水災也能預測吧，那土石流應該也可以，就可以再發生前趕快逃跑。

全文:https://ec.ltn.com.tw/article/breakingnews/3201082

**Comment by pocper1** [comment]
在台灣如果可以應用這套系統，暑假期間發生許多河水快速上升造成人被沖走的事件，就可以避免了。

**Comment by Timmy110403540** [comment]
這項科技正好可以面對全球暖化水位上升的現象，及早讓低窪地區的人做準備。

**Comment by bonny123865** [comment]
兩家公司的合作可以為更多用戶對於分析服務更容易取得，而且還跨了各種不同領域的產業，非常強！

**Comment by briankuo123** [comment]
台灣的降雨量非常驚人，河道坡度也很大，如果能夠提前預測就能夠為那些住在高風險區的人爭取更多時間。

**Comment by ghost** [comment]
水災能夠預測的話 或許哪天連地震都能了呢~
